webplugin/js/app/kommunicate.js

```diff
@@ -916,4 +916,31 @@ $applozic.extend(true, Kommunicate, {
             }
         }
     },
+    getUserLocation:async function (){
+        if (!("geolocation" in navigator)) {
+          console.error("Geolocation is not supported by this browser.");
+          return "Geolocation not supported";
+        }
+      
+        const getCurrentPosition = () => 
+          new Promise((resolve, reject) => 
```

Comment:
We already have the `latitude` and `longitude` global variables use those variables.